lua/caramba/chat.lua

```
@@ -343,104 +374,70 @@ M._send_message_with_context = function(cleaned_message, contexts, search_result
+        content = content,
+        plan = plan, -- Store for later execution
+      })
+      M._render_chat()
     end)
   end)
 end
 
+-- Format a plan to be displayed in the chat window
+M._format_plan_for_chat = function(plan, review)
+  local lines = {}
+  
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)